test/addons/callback-scope/test-resolve-async.js

```diff
@@ -5,9 +5,6 @@ const assert = require('assert');
 const { testResolveAsync } = require(`./build/${common.buildType}/binding`);
 
 let called = false;
-testResolveAsync().then(common.mustCall(() => {
-  called = true;
-}));
+testResolveAsync().then(() => (called = true));
```

Comment:
Any reason for the parentheses here instead of the seemingly-more-idiomatic curly braces?